migrations/tools/abi.ts

```diff
@@ -0,0 +1,16 @@
+import { Interface } from "ethers/lib/utils";
```

Comment:
This file can be removed and imported from `@skalenetwork/upgrade-tools`.

Reply:
Agree